### PR TITLE
Fixes for log replication

### PR DESCRIPTION
**Pull request by garrensmith**
One small fix to the readme. Then I ran the test with leveldown and realized that they didn't always work. There were some race conditions. So I've make fixes for that as well as changed the tests so they run against memdown and leveldown.

### CODE REVIEW

File: log.js [hunk 1]
```diff
@@ -24,7 +26,19 @@ class Log {
   constructor (node, {adapter = require('leveldown'), path = ''}) {
     this.node = node;
     this.committedIndex = 0;
-    this.db = levelup(encode(adapter(path), { valueEncoding: 'json', keyEncoding: 'number'}));
+    this.db = levelup(encode(adapter(path), { valueEncoding: 'json', keyEncoding: 'binary'}));
+    this._dbQueue = [];
+  }
+
+  dbTask (fun) {
```
Comment (3rd-Eden) [inline log.js]: Needs JSDoc

File: log.js [hunk 2]
```diff
@@ -10,6 +10,8 @@ const encode = require('encoding-down');
  * @property {object}  command The command to be used in the raft state machine
  */
 
+ let isWriting = false;
```
Comment (3rd-Eden) [inline log.js]: Is this a debug left over? Doesn't seem to be used in this PR>

File: README.md [hunk 3]
```diff
@@ -31,7 +31,7 @@ npm install --save liferaft
   - [LifeRaft#leave()](#liferaftleaveaddress)
   - [LifeRaft#promote()](#liferaftpromote)
   - [LifeRaft#end()](#liferaftend)
-  - [LifeRaft#saveCommand()](#liferaftsavecommand)
+  - [LifeRaft#ommand()](#liferaftcommand)
```
Comment (3rd-Eden) [inline README.md]: LifeRaft#command instead of ommand

**Comment by garrensmith** [comment]
Sorry @3rd-Eden, I had a bit of a bad commit and some things got left in that shouldn't have. I've cleaned up the PR. Thanks for reviewing so quickly.

**Comment by 3rd-Eden** [comment]
Releasing this as the next major.

**Comment by garrensmith** [comment]
Awesome thanks @3rd-Eden

**Pull request merged**